operator/setappointee.go

```diff
@@ -0,0 +1,63 @@
+package operator
+
+import (
+	"context"
+
+	allocationManager "github.com/Layr-Labs/eigensdk-go/contracts/bindings/AllocationManager"
+	mockAvsServiceManager "github.com/Layr-Labs/eigensdk-go/contracts/bindings/MockAvsServiceManager"
+	"github.com/Layr-Labs/eigensdk-go/utils"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+func (o *Operator) SetAppointee(instantSlasherAddr common.Address, serviceManagerAddr common.Address,allocationManagerAddr common.Address,registryCoordinatorAddr common.Address) error {
```

Comment:
We should do this from the deployment script.

Reply:
I don't think users will look at the scripts, they would directly look at the offchain code imo. And scripts really don't tell which order to call the particular functions

Reply:
moved to deployment scripts, kept these also to use in cli